src/libponyc/verify/fun.c

```diff
@@ -1,8 +1,136 @@
 #include "fun.h"
+#include "../type/lookup.h"
 #include "../type/subtype.h"
 #include "ponyassert.h"
 #include <string.h>
 
+static bool verify_calls(pass_opt_t* opt, ast_t* ast)
```

Comment:
Could this be named something less generic than `verify_calls`? It is very specific to the `runtime_override_defaults` method, but that isn't clear at all from the name.

Reply:
done